assets/stylesheets/bookingsync/_ember-toggle.scss

```diff
@@ -7,3 +7,7 @@
 .x-toggle:checked + label > .x-toggle-light.x-toggle-btn {
   background: $brand-primary;
 }
+
+.bsy-switch {
```

Comment:
Why do we need this? :)

Reply:
without that styling is broken, switch can't be inline with other elements.